plugins/linguist-backend/api-report.md

```diff
@@ -96,6 +98,8 @@ export interface RouterOptions {
   reader: UrlReader;
   // (undocumented)
   scheduler?: PluginTaskScheduler;
+  // (undocumented)
+  tokenManager: TokenManager;
```

Comment:
This is technically a breaking change, so you could either make it optional (and handle the complexity of that in code), or make the changeset `minor` level. The changeset should also probably say explicitly that it added a token manager as the dependency to the plugin, so as to be informative to the reader who wonders what the effect is on them as they are upgrading.

Reply:
I've marked the change as "breaking". In the install instructions all of the "PluginEnvironment" is spread as the RouterOptions so there will probably be no practical impact when upgrading.

Reply:
Yeah it'll depend on people's individual setups but in practice most probably have followed those.